src/ui-mapbox/index.android.ts

```diff
@@ -1223,23 +1222,18 @@ export class Mapbox extends MapboxCommon implements MapboxApi {
         });
     }
 
-    async addImage(imageId: string, image: string, nativeMap?: any): Promise<void> {
-        return new Promise((resolve, reject) => {
+    async addImage(imageId: string, imagePath: string, nativeMap?: any): Promise<void> {
+        return new Promise(async (resolve, reject) => {
             const theMap = nativeMap || this._mapboxMapInstance;
 
             if (!theMap) {
                 reject('No map has been loaded');
                 return;
             }
 
-            if (!image.startsWith('res://')) {
-                image = path.join(knownFolders.currentApp().path, image.replace('~/', ''));
```

Comment:
This replacement of `~/` is already part of the nativescript library: https://github.com/NativeScript/NativeScript/blob/main/packages/core/image-source/index.android.ts#L120